src/models/bgp/attributes/mod.rs

```diff
@@ -5,7 +5,12 @@ mod nlri;
 mod origin;
```

Comment:
A unit-test for the new attributes capabilities could be nice to add here, but I'm sure attributes will be tested elsewhere too, so not a big concern.

Reply:
Are there any specific changes that you want unit tests to cover?